### PR TITLE
Update libraries.mdx

### PR DESCRIPTION
**Pull request by cmoulliard**
- Includes the Java Quarkus Vault project

### CODE REVIEW

File: website/content/api-docs/libraries.mdx
Comment (VioletHynes) [inline website/content/api-docs/libraries.mdx:115]:
```suggestion
- [Quarkus Vault](https://github.com/quarkiverse/quarkus-vault)
```

**Review by VioletHynes — COMMENTED** [review]
Hey there! I know this has lingered for a while, but I don't see why we can't merge this as long as Vault gets capitalized as suggested. Thanks for the contribution, and patience.

**Review by yhyakuna — APPROVED** [review]

**Comment by VioletHynes** [comment]
Hey @cmoulliard ! Thanks for accepting the suggestion. I don't seem to have permission to merge main into this branch, which it looks like we'll need to get CI working (since this has pretty old GHAs right now). Could you do so? Once everything passes and looks good, I'm more than happy to merge this. Thanks for the contribution, and I'm sorry it took us a while to get to this!

**Comment by cmoulliard** [comment]
> Could you do so?

No as I don't have the rights (= not authorized) to merge it

**Pull request merged**
